tests/test_overload_handler.py

```
@@ -1,5 +1,6 @@
 """Test the overload handler."""
 
+import time
```

Ruff (0.8.2)

`time` imported but unused

Remove unused import: `time`

(F401)